tests/test_newick.py

```diff
@@ -353,15 +359,15 @@ def test_n2_example(self):
     def test_n4_example(self):
         # $ mspms 4 1 -T -r 4 10 -p 8
         msout = """
```

Comment:
I'd rather ``textwrap.dedent()`` stuff like this rather than wrapping it up against the file. It just makes everything unreadable

Reply:
Fixed by turning off line length warning.